### PR TITLE
CPRI-4157 Moved toggle section to parent component

### PR DESCRIPTION
**Pull request by siddharth-pearson**

### CODE REVIEW

**Comment by propattern** [comment]
Aren't we writing any unit tests for the components? @devu

**Comment by devu** [comment]
We do and we should, but yeah coverage is 'minimal' at the moment. I could forge it for now as it under heavy refactoring but once we working on final shape of it we should fully cover it

File: src/components/AssetPlanRequest/AssetPlanRequestContainer.js [hunk 1]
```diff
@@ -56,8 +57,14 @@ class AssetPlanRequestContainer extends React.PureComponent {
     } = this.props
     const { renderFormField } = this.uiFactory
     const allExpanded = sourceInfoExpanded && placementExpanded && usageExpanded
-    const collapseSections = () => actions.collapseAll()
-    const expandSections = () => actions.expandAll()
+    const collapseAllSections = () => actions.collapseAll()
+    const expandAllSections = () => actions.expandAll()
+    const toggleSection = (sectionID) => actions.expandSection({
```
Comment (devu) [inline src/components/AssetPlanRequest/AssetPlanRequestContainer.js]: this doesn't make sense to me to be honest. You move this toogle all the way and put in nested ternary, while every section had direct access via it's own container to the prop they need?

**Comment by devu** [comment]
Rejected, this is reverting back to non scaleable solution, each section has it's own container to get the props it needs from store

**Pull request closed without merging**